### PR TITLE
Update `Dialog` component

### PR DESCRIPTION
**Pull request by PavelHolec**

### CODE REVIEW

File: Sources/OrbitStorybook/Detail/Items/StorybookDialog.swift [hunk 1]
```diff
@@ -72,6 +62,7 @@ struct StorybookDialogPreviews: PreviewProvider {
     static var previews: some View {
         OrbitPreviewWrapper {
             StorybookDialog.basic
+            StorybookDialog.normal
```
Comment (sjavora) [inline Sources/OrbitStorybook/Detail/Items/StorybookDialog.swift:65]: `basic` **and** `normal`? 🤔

File: Sources/Orbit/Components/Dialog.swift [hunk 2]
```diff
@@ -77,21 +78,43 @@ public struct Dialog: View {
...
+
+    /// Creates Orbit Dialog component.
+    public init(
+        illustration: Illustration.Image = .none,
+        title: String = "",
+        description: String = "",
+        style: Style = .primary,
+        buttons: Buttons,
+        descriptionLinkAction: @escaping TextLink.Action = { _, _ in }
+    ) where Content == EmptyView {
```
Comment (sjavora) [inline Sources/Orbit/Components/Dialog.swift:108]: Why not set the default content value in the other init?

```swift
@ViewBuilder content: () -> Content = { EmptyView() }
```

That way, this init wouldn't have to be written.

Comment (PavelHolec) [reply Sources/Orbit/Components/Dialog.swift:108]: Forgot we can finally do that

File: Sources/OrbitStorybook/Detail/Items/StorybookDialog.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/OrbitStorybook/Detail/Items/StorybookDialog.swift:65]: This is using the existing naming (basic stands for the tab content and the normal is used within the component preview)

**Review by sjavora — APPROVED** [review]

**Pull request merged**